HW1/P5/P5_connected_components.py

```diff
@@ -0,0 +1,25 @@
+from P5_bfs import *
```

Comment:
For problems like this one, it would be helpful to have comments to give insight on your strategy